pkg/joinserver/grpc_application_activation_settings_registry.go

```diff
@@ -76,22 +82,8 @@ func (srv applicationActivationSettingsRegistryServer) Set(ctx context.Context,
 		)
 	}
 	v, err := srv.JS.applicationActivationSettings.SetByID(ctx, req.ApplicationIdentifiers, req.FieldMask.Paths, func(stored *ttnpb.ApplicationActivationSettings) (*ttnpb.ApplicationActivationSettings, []string, error) {
-		if stored == nil {
-			if err := ttnpb.RequireFields(req.FieldMask.Paths,
-				"kek_label",
-			); err != nil {
-				return nil, nil, errInvalidFieldMask.WithCause(err)
-			}
-			if !ttnpb.HasAnyField(req.FieldMask.Paths, "kek.key") ||
```

Comment:
`KEK` was marked as required in original issue, did that requirement change?

Reply:
See above